webnn/resources/utils.js

```diff
@@ -291,6 +291,7 @@ const PrecisionMetrics = {
   batchNormalization: {ULP: {float32: 6, float16: 6}},
   clamp: {ULP: {float32: 0, float16: 0}},
   concat: {ULP: {float32: 0, float16: 0}},
+  constant: {ULP: {float32: 2, float16: 2, int32: 0, uint32: 0, int64: 0, int8: 0, uint8: 0}},
```

Comment:
This is an interesting case because there are two `constant` overloads, with one that should be always *exact* (the one taking the Float32Array), but the other with start and step could have a ULP delta of 2.

Technically the integers could be off by 1 too for large values too (above 16777216) since the `constant` overload only accepts float start and step values, but none of our test cases go near that high. So in the general case, 0 ULP is valid for integers, and that's fine.

Reply:
(feel free to resolve this comment)